control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -3496,7 +3531,7 @@ func (r *HostedControlPlaneReconciler) reconcileOperatorLifecycleManager(ctx con
 				}); err != nil {
 					return fmt.Errorf("failed to reconcile catalogs image stream: %w", err)
 				}
-			} else {
+			} else if managedService != hyperv1.AroHCP {
```

Comment:
I'd prefer to rely on a capability here. If the platform does not support imagestreams, then don't try to delete them.

Reply:
Thanks, it's been updated!